sdk/monitor/monitor-query/src/internal/logQueryOptionUtils.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LogsQueryClientOptions } from "../logsQueryClient";
+
+export function getLogQueryEndpoint(options: LogsQueryClientOptions): string {
+  if (!options.endpoint) {
+    throw new Error("options.endpoint is required");
+  }
+
+  const url = new URL(options.endpoint);
+  url.pathname += "v1";
```

Comment:
is it worth doing `url.pathname = "/v1"` in case they mistakenly pass in something else?